sample/src/main/java/com/canhub/cropper/sample/options_dialog/app/SOptionsDialogBottomSheet.kt

```diff
@@ -189,8 +188,10 @@ internal class SOptionsDialogBottomSheet : BottomSheetDialogFragment(), SOptions
         when (options.cropShape) {
             CropImageView.CropShape.RECTANGLE -> binding.cropShape.chipRectangle.isChecked = true
             CropImageView.CropShape.OVAL -> binding.cropShape.chipOval.isChecked = true
-            CropImageView.CropShape.RECTANGLE_VERTICAL_ONLY -> binding.cropShape.chipRectangleVerticalOnly.isChecked = true
-            CropImageView.CropShape.RECTANGLE_HORIZONTAL_ONLY -> binding.cropShape.chipRectangleHorizontalOnly.isChecked = true
+            CropImageView.CropShape.RECTANGLE_VERTICAL_ONLY -> binding.cropShape.chipRectangleVerticalOnly.isChecked =
+                true
+            CropImageView.CropShape.RECTANGLE_HORIZONTAL_ONLY -> binding.cropShape.chipRectangleHorizontalOnly.isChecked =
+                true
```

Comment:
This changes broke the ktlintCI

Reply:
How so? Why did it break anything? What is the change here, anyway? I don't see any error here on the IDE... The sample works fine. Everything builds fine...

Reply:
Anyway, I've made it have a new line as the logs that you've shown say. Still a very weird issue.

Reply:
The error message:

```
> Task :sample:ktlintMainSourceSetCheck FAILED
/home/runner/work/Android-Image-Cropper/Android-Image-Cropper/sample/src/main/java/com/canhub/cropper/sample/options_dialog/app/SOptionsDialogBottomSheet.kt:191:62 Missing newline after "->"
/home/runner/work/Android-Image-Cropper/Android-Image-Cropper/sample/src/main/java/com/canhub/cropper/sample/options_dialog/app/SOptionsDialogBottomSheet.kt:193:64 Missing newline after "->"
```

**The Ci link:** https://github.com/CanHub/Android-Image-Cropper/pull/104/checks?check_run_id=2239558660


![Screenshot 2021-04-01 at 09 51 59](https://user-images.githubusercontent.com/15942449/113269027-f3bbf880-92cf-11eb-9892-9bbaeecae2ed.png)

Reply:
It shows it even though I've added a new line now?

Reply:
Now is ok, you can see on the bottom of the github PR page =D

Reply:
How did you perform this check? How come this issue even exists, while the IDE is completely fine with it?